apps/quickstart-react-ts/src/middlewares/authentication.ts

```diff
@@ -0,0 +1,19 @@
+import jwt from 'jsonwebtoken';
+const SHARED_SECRET = process.env.MONDAY_SIGNING_SECRET;
+
+export default async function authenticationMiddleware(req, res, next) {
+  try {
+    let { authorization } = req.headers;
+    if (!authorization && req.query) {
+      authorization = req.query.token;
+    }
```

Comment:
```suggestion
    const authorization = req.headers.authorization || req?.query?.token;
```

Reply:
did something in the same spirit, also made changes to return more accurate errors